packages/geoprocessing/src/toolbox/geoblaze/rasterStats.ts

```diff
@@ -25,12 +25,16 @@ export interface RasterStatsOptions extends CalcStatsOptions {
   filterFn?: (cellValue: number) => boolean;
   /** Optional number of bands in the raster, defaults to 1, used to initialize zero values */
   numBands?: number;
+  /** For categorical rasters, category is equal to the classId in metricGroup */
+  categorical?: boolean;
+  categoryClassValues?: string[];
```

Comment:
Looks like the param comments just need a little touch-up, nice work getting this in.